XenAdmin/Properties/AssemblyInfo.cs

```diff
@@ -35,10 +35,10 @@
 // General Information about an assembly is controlled through the following 
 // set of attributes. Change these attribute values to modify the information
 // associated with an assembly.
-[assembly: AssemblyTitle("[XenCenter]")]
-[assembly: AssemblyDescription("[XenCenter]")]
+[assembly: AssemblyTitle("XenCenter")]
+[assembly: AssemblyDescription("XenCenter")]
 [assembly: AssemblyConfiguration("")]
-[assembly: AssemblyProduct("[XenCenter]")]
+[assembly: AssemblyProduct("XenCenter")]
 

```

Comment:
Looks like these are leftover from building XenCenter with branding? 👀 We should revert them. Same for the value in `XenAdmin.csproj`